rpcs3/Emu/RSX/RSXFIFO.cpp

```diff
@@ -56,9 +58,10 @@ namespace rsx
 			}
 		}
 
+		template <bool TestEqual>
 		void FIFO_control::set_get(u32 get)
 		{
-			if (m_ctrl->get == get)
+			if (TestEqual && m_ctrl->get == get)
```

Comment:
why not if constexpr?

Reply:
It's more lines of code and compiler's generated is the same. (it means separation of `if constexpr (TestEqual)` and `if (m_ctrl->get == get)`)